### DISCUSSION TITLE
Akash Community Marketing/Support Team Proposal

### DISCUSSION THREAD

**Discussion by piber-dev** [akash-network/community#756] [Governance Proposals]
# Akash Community Marketing/Support Team Proposal

## Introduction

After seeing many great features being released on Akash such as GPUs, easy-to-use CI/CD workflows, and fiat/trial funds, we feel like the network is ready for the masses. Now is the time to start marketing and supporting the network, community, platforms and tools, on a larger scale. We want to explore creating a focused and skilled marketing/support team for Akash helping users using the network and bringing users to the network. Examples include: posts on X/Reddit/Medium, sponsored videos where Akash is used, sponsoring small deployments in exchange for banners (game servers, web tools), helping create integrations to Akash, and creating smaller programs within the program such as "Deployment/Contribution of the month." 

## Proposal Research

This document was mentioned on the December Steering Committee Call and presented in front of the SIG Community Call after that. We have also set up groups and calls with Overclock Labs employees, as many as we could reach that were relevant to this proposal. So far, the feedback has only been good. Support on different social channels is something Adam (Head of Community) agrees can be improved upon. Furthermore, we have received a great amount of feedback from the [Decloud Nodes Lab Validator](https://declab.pro/) which we have worked to incorporate into this document. The hourly funding for this proposal is similar to other proposals in the past.

Big shoutout to AndrewG (Decloud Nodes Lab) for the great feedback!

## Strategies

Please refer to [this page](https://docs.google.com/spreadsheets/d/1wXMM8lB4j2E-k48Nds_xE1Och46uPzIInCUY4wTww3U/edit?gid=961413619#gid=961413619) for a full breakdown of different strategies to be delivered, and [this page](https://docs.google.com/spreadsheets/d/1wXMM8lB4j2E-k48Nds_xE1Och46uPzIInCUY4wTww3U/edit?gid=1606194421#gid=1606194421) for a breakdown of experimental (and free) strategies to potentially be used and delivered. This page also includes some of the results we expect this proposal to have. Many results will only be published during the Q1 result and not predicted in beforehand and if you want some information from any Q4 item please leave a comment here.

### Notable strategies from both the paid and free/experimental lists

- Creating posts on different platforms (X, Reddit, Medium) with different educational, promotional, and informative text from our own personal channels or new channels created for this proposal. Examples include: conducting thought experiments in regard to creating images and deployments, making comparisons to e.g. other tech, creating new templates for communities to use, stress testing the network, and more. The goal of this is to make Akash something people recognize over time and help out in other communities where there is no good standardized way of running their tools.
- Attempting to work with the Events 2025 Proposal Teams to educate attending team members on how to be a useful events participant. Examples include: creating documents on how to collect leads, funnel users to different community programs, and executing marketing strategies. In case the marketing team does attend any events, we will educate ourselves on how to improve ourselves within the roles from the [attendees examples for the Events 2025 Proposal](https://docs.google.com/spreadsheets/d/1DBXUhNsf8qxIYD-NtuLeGXhmQ-WTOdcCfdyvXgaGGV0/edit?gid=580542914#gid=580542914&range=B26:E29).
- Bringing back user support for our communities on Reddit and X, at least once every workday. Some of the communities have been neglected for years now, it is time this changes. While we have many users commenting on posts on e.g. Reddit, it is good to have people with up-to-date information also responding when it makes sense to do so.
- Helping different platforms promote their new features that can be used by the masses (e.g. Akash Console with their recent CI/CD tool once this is out of beta, or the credit card integration) through sponsoring a relevant YouTube channel to create a video on it. The goal of this is to increase the number of unique developers deploying their own applications on Akash.
- Bringing back our presence on X spaces by aiming to host or getting invited to different meetings at least once per month. Each meeting will have different goals.
- Helping write a monthly newsletter for Akash including an overview of what is being worked on, relevant events, updates, and programs.
- And please be aware that we are open for ideas to add to the non-billed experimental tasks list for strategies utilizing the miscellaneous marketing budget.

> Note: while we would like to complete everything, we offer the experimental tasks for free both both because we don't know how many hours to expect from these and that we are not certain that we can complete all of them. Collaboration with the core team is crucial to get e.g. the Events 2025 collaboration delivered. It is also our first time doing some of these experimental strategies, and things can go wrong - don't expect all of the experimental tasks to be completed.

## Resources required

For a detailed breakdown of the resources required related to funding, please refer to [this page](https://docs.google.com/spreadsheets/d/1wXMM8lB4j2E-k48Nds_xE1Och46uPzIInCUY4wTww3U/edit?usp=sharing).

> 4x contributors, to be scaled up as time goes on.
> 
> A marketing budget where we are not limited to a tight budget
>
> The possibility of collaborating with other proposals and teams. While this is only required for the experimental tasks, we are hoping for experimental tasks to become main reoccuring tasks in Q2.

The main contributors for the first proposal are: Rodri (technical role and responsible for financials), PiberDev (program lead and technical role), and Robert (non-technical role and social media advisor). All of these contributors have attended events representing Akash in the past, and all of these contributors are also long-term Insiders/Vanguards. We are also welcoming Lil D, an Akash Insider, who will be starting out helping with the newsletters and recaps.

As detailed in the budget breakdown, tasks are broken down into hours and frequency. This divides the contributor pay depending on how much work each contributor works. Work 5 hours, get 5 hours of pay. Work 20 hours, get 20 hours of pay. Not everyone has the same skillsets or availability and this is crucial when the program scales in the future.

## Proposal funding

We are requesting 3 months worth of funding for Q1 2025, totaling $34 967,50 USD worth of AKT. About a third of the currently expected hours we expect this program to require will be given for free in this proposal while we run this Q1 pilot. No additional overages will be included but contributor pays will be prioritized and based in USD at the time of paying them in case of volatility. If this proposal passes, we will hold on to the budget for Q1 and send it back if the proposal for Q2 does not pass. The proposal for Q2 may not request as much if there is overage from this proposal that can be subtracted from the Q2 proposal's request. If we do not believe we are getting good results on the marketing budget, we will try to make any appropriate changes on the experiments to get better results.

### Proposal Date

The proposal is set to go live next week, to be voted on during next week and during the holidays. Please go through this proposal and give feedback before then.

**Comment by ghost** [comment]
WIsh you the best to you and all parties involved in this proposal, decentralized cripto marketing is the next step in the decentralized Akash community

**Reply by piber-dev** [reply]
thanks @instafinanzas! would love to continue working with you through this proposal if it passes.

**Comment by nick134-bit** [comment]
GM,
Currently, the Vanguards seem to be responsible for support throughout all channels, and this will be included in the community spend https://github.com/orgs/akash-network/discussions/753 already.
It's also planned to have bounties for content creation like videos which would be open to anyone. These are also covered in the proposal mentioned before. One goal of these bounties is also to democratize contribution opportunities, and this proposal feels like a step back to me because it eliminates room for external contributions.
While I don't oppose the idea of improving support and intrinsic marketing, I think this should be funded via already existing sources or a more action-based approach like bounties.

**Reply by piber-dev** [reply]
gm @nick134-bit - thanks for your comment! please read through my response and let me know if this changes your view of this proposal:

> Currently, the Vanguards seem to be responsible for support throughout all channels, and this will be included in the community spend https://github.com/orgs/akash-network/discussions/753 already.

while the vanguards program might expand, this is not yet set in stone. there was some internal vanguards discussion to have a reddit/twitter strategy after i brought it up, and we thought we would do this quite soon so we originally removed at least the reddit support from this document when presenting in front of the SIG community call last week. however, adam (head of community, also the one responsible for the vanguards program) thought we should keep it at least for the first quarter as nothing is set in stone yet and the support on both these platforms have been lacking.

> It's also planned to have bounties for content creation like videos which would be open to anyone. These are also covered in the proposal mentioned before. One goal of these bounties is also to democratize contribution opportunities, and this proposal feels like a step back to me because it eliminates room for external contributions.

this is a great point! the issue we've seen is that it's hard to get involved even though you're actively trying, so this program will act as a duct tape letting us get more involved until the other programs develop more. maybe this proposal will not be needed if the other programs get mature enough in the future, but we feel like there is a need to do this ourselves for the time being as we feel like multiple aspects of the network don't get enough attention and could need more people working on it. 

when i calculated how much the community pool is growing, i estimated it to be $49 million USD per year at a $3.80 token price and 13% inflation. there is a lot of room to have a lot of initiatives in place on the network. this can support a lot of developers, even if a single program may be unable to do so. we believe the current programs can't support enough part/full-time developers, and neither will this program. but as mentioned, the community pool is large and there can be a lot of proposals made by a lot of teams/users.

i can't stress it enough that we really want the other programs by overclock to become better but this takes time for them. in the next insiders call, we might have time to have some discussion on how to improve the structure of the programs to be more community led and less centralized to overclock labs. other improvements also seem like they are coming, but nothing is planned yet so in my opinion there is a need for this proposal at the very least for the first quarter. then another assessment can be made in Q2.

> While I don't oppose the idea of improving support and intrinsic marketing, I think this should be funded via already existing sources or a more action-based approach like bounties.

the bounties are great but these are also limited in how much people can get involved with and what can be given. the zealy pilot program was probably the only time budget has been given on top of rewards for completing the bounty so this would make it impossible to e.g. sponsor youtube videos. i've also been a little active in the website repository and there is really not a lot of work to get involved with and when there is, there is a lot of competition and usually the commenting back and fourth takes longer time than the task itself. if we could get involved with the current programs and options, we would, but it's practically impossible right now to be a full time community member on akash without creating larger proposals.

**Reply by nick134-bit** [reply]
> while the vanguards program might expand, this is not yet set in stone. there was some internal vanguards discussion to have a reddit/twitter strategy after i brought it up, and we thought we would do this quite soon so we originally removed at least the reddit support from this document when presenting in front of the SIG community call last week.

I would suggest that this proposal focus on expanding the Vanguards program, rather than creating a new capsule to address a problem that should be solved by an existing entity. In my opinion, this proposal would unnecessarily complicate and obscure the organizational structure, making it less efficient.

> multiple aspects of the network don't get enough attention and could need more people working on it.

I completely agree, but does this proposal really add more people? As far as I know, 3/4 of the proposed members are already Vanguards who receive some form of compensation for their work in this field, so it is unclear whether this proposal would actually increase the number of people working on these issues.

> when i calculated how much the community pool is growing, i estimated it to be $49 million USD per year at a $3.80 token price and 13% inflation. there is a lot of room to have a lot of initiatives in place on the network. this can support a lot of developers, even if a single program may be unable to do so. we believe the current programs can't support enough part/full-time developers, and neither will this program. but as mentioned, the community pool is large and there can be a lot of proposals made by a lot of teams/users.

I do not see the relevance of this information. Simply because the community pool has the potential to support numerous initiatives does not mean that it is necessary to allocate funds to this specific proposal.
On another note this initiative would cost almost exactly the same as the whole Vanguards program (35K instead of 36K) per quarter in its base version with 4 people, while the Vanguards support up to 12 people with the current budget, and I'm not able to see comparable benefits to the network.

> we might have time to have some discussion on how to improve the structure of the programs to be more community led and less centralized to overclock labs. 

I did not intend to imply that Overclock Labs is the centralized entity in question. Rather, I am concerned that a small group of community members would encapsulate a problem that should be addressed at a different level (Vanguards), creating an unnecessary layer of bureaucracy. This extra layer makes it easier for the team members of this proposal to contribute and get paid, but will obstruct any other community member from contributing to the same really broad field because funds are already allocated.

**Reply by piber-dev** [reply]
thanks @nick134-bit for your updated response! while i appreciate the updated message, i don't think you completely understand what we are proposing yet. i tried to give you some perspective on why some of the things you want are not possible and that does not have to be a permanent proposal, but you still seem to think that it's as easy as to just modify other people's proposals. our experience tell a different story i'm afraid, and we are hoping that our proposal can change that over time. but we really see the need for this proposal until that time comes.

> I would suggest that this proposal focus on expanding the Vanguards program, rather than creating a new capsule to address a problem that should be solved by an existing entity. In my opinion, this proposal would unnecessarily complicate and obscure the organizational structure, making it less efficient.

please post this feedback [here](https://github.com/orgs/akash-network/discussions/753) if you think this is outside the scope of a new proposal. while we have ideas for the vanguards program we would like to see, we have no control over adam's proposal more than the feedback we are trying to give on it. but for now, it sounds like [adam does not want to change the vanguards program while the voting is happening soon](https://github.com/orgs/akash-network/discussions/753#discussioncomment-11549591).

> I completely agree, but does this proposal really add more people? As far as I know, 3/4 of the proposed members are already Vanguards who receive some form of compensation for their work in this field, so it is unclear whether this proposal would actually increase the number of people working on these issues.

the proposal is larger than adding reddit/twitter support and nothing is of a duplicated effort. what do you mean with "this field" and that it would not increase the number of people working on these issues? we are talking about going from zero people to a couple people for most issues, and we are not being paid for the listed issues currently. while we have vanguards on the team, the only relevance that has to this proposal is that we have experience supporting users on various technical and general questions in my opinion.

> I do not see the relevance of this information. Simply because the community pool has the potential to support numerous initiatives does not mean that it is necessary to allocate funds to this specific proposal.

you're right that it is not necessary to fund this specific proposal, i just wanted to mention that anyone can create their own proposal so just because one team is behind one proposal does not mean that the same efforts can't be made by other people. we are not making it harder for other community members to get involved, at worst it will remain the same and the best case scenario is that we can onboard more people in future proposals or that the success of this proposal might lead to other programs to change, such as the vanguards one.

> On another note this initiative would cost almost exactly the same as the whole Vanguards program (35K instead of 36K) per quarter in its base version with 4 people, while the Vanguards support up to 12 people with the current budget, and I'm not able to see comparable benefits to the network.

did you see the program breakdown? for the reddit/twitter support alone, we are looking at a budget of $6825 for the entirety of Q1, so essentially we are supporting two platforms at less than 20% of the price of the vanguards proposal. reddit and twitter are platforms that are not being supported enough, but we don't believe that it needs the same amount of attention as discord and we are not asking for that either. a lot of the budget is for tangible marketing results only, such as views, sponsored deployments, rewards pool for successful contests (gamejams, "deployment of the month," etc). 

you are also comparing the vanguards proposal budget with this budget, but much of the work of vanguards is not active work making the hourly pay low for that proposal for that reason. the goal is to have response times of max 1 hour on messages, which means that for low activity periods vanguards can multitask a lot while doing the support. we also have a lot of items we are including in our budget, much more than only support. this is why it can't just be an extension of the vanguards proposal, though the support part of it could definitely be an expansion of the program in Q2 and at the same time removed from this proposal.

> I did not intend to imply that Overclock Labs is the centralized entity in question. Rather, I am concerned that a small group of community members would encapsulate a problem that should be addressed at a different level (Vanguards), creating an unnecessary layer of bureaucracy. This extra layer makes it easier for the team members of this proposal to contribute and get paid, but will obstruct any other community member from contributing to the same really broad field because funds are already allocated.

we agree that some of the items can be solved at a different level, but since the items are not solved currently we want to work on those items for the time being. if this changes, we will remove any unnecessary items and try to get involved with those programs instead. as mentioned, this does not make it harder for other users to get involved, it makes it easier for us to get involved and if the proposal is scalable we want to do that. much of this proposal is a duct tape solution and we are actually hoping for other proposals to be easier to get involved with but for the last year we have tried to get involved by just throwing feedback at OCL but have not been able to push things through.

**Comment by rodri-r** [comment]
Thank you for creating the proposal @piber-dev 

As we talked about this in several calls between the suggested team, with Adam, with other community members, in SIG community call and in some Discord channels, most of us agree that we are missing some marketing and presence on several networks. (ex: Reddit, X spaces, newsletter) especially some tech support in some social networks besides Discord, which IMO should draw more attention to Akash and help onboard more users and developers.  Not everyone uses Discord and as Piber mentions "this program will act as a duct tape letting us get more involved until the other programs develop more".

While I support this proposal, I will vote ABSTAIN if this goes on chain, as I am part of the suggested team and want to avoid any conflict of interest.

**Reply by piber-dev** [reply]
totally right - thanks for writing and thanks for the support @rodri-r!

**Comment by Aihonaken** [comment]
Hey guys! Thnx for the proposal, i think technical support is really needed in Twitter and Reddit. It's time to boost our presence, and draw some attention. I will support this proposal, as i think by the end of the pilot we have deliverables we can track and check if the project is a success or we need to revamp it.

**Reply by piber-dev** [reply]
thanks for your support @Aihonaken !! i totally agree with everything

**Comment by hewi333** [comment]
Hey @piber-dev thanks for taking the time to identify areas where we can progress the network. I like how you've called out experimental tasks for ideation but are only billing hours to core tasks.

My big question to the group is what are the handles/usernames you will be posting & supporting from? Will these all be personal accounts, or are you trying to spin up dedicated "Akash Community" accounts?

Also, in the tasks, you call out there will be posts on X, Reddit, and Medium. Can you provide some examples of content that has been published on these channels for voters to assess quality?

**Reply by piber-dev** [reply]
hey @heyhewi thanks for your comment - fun seeing someone who also likes the experimental tasks!

> My big question to the group is what are the handles/usernames you will be posting & supporting from? Will these all be personal accounts, or are you trying to spin up dedicated "Akash Community" accounts?

i received the same question yesterday privately and unless access to the official channels in some capacity are offered to us, we have no plans of using/needing them for this proposal. depending on what is required to collaborate on e.g. medium/reddit posts, we might have to set up community accounts but we don't feel the need to set them up just for the sake of doing so but if this is a preference by the community we can set them up for sure!

it sounded yesterday in a private meeting that we might be able to participate in X spaces to the ones hosted by the "official account" from our personal accounts because it would be clear that there are community members speaking and not overclock labs employees speaking. but this is not something that we require for this proposal. if this changes or is not possible, we can set it up ourselves.

> Also, in the tasks, you call out there will be posts on X, Reddit, and Medium. Can you provide some examples of content that has been published on these channels for voters to assess quality?

we are going to create educational and promotional posts for these platforms and while we do have examples posted by some of the members of the team already, we want to increase the quality even more. some of the promotional posts will include personal experiences from e.g. being involved in a certain program. [rodri's medium](https://medium.com/@rodri_), [my python/akash workshop](https://github.com/piber-dev/python-deployment-workshop), and [rodri's X page](https://x.com/rodrigo_asterix/) are some good examples. we have also started filling out the Q4 tracking on the google sheets page but we are not making this a big priority when we have a lot of other stuff with pushing this proposal currently.

**Reply by rodri-r** [reply]
Thanks for your comment @heyhewi to add a bit more to what @piber-dev wrote and maybe help answer your questions:

Here are a couple of links to recent threads that I created about Akash in my personal X account:

https://x.com/rodrigo_asterix/status/1863852631883477404

https://x.com/rodrigo_asterix/status/1867620553764311533

Wanted to highlight them, as I also share memes and other stuff in my X account.  Reason as well, why I think that creating some "dedicated Akash Community accounts" specifically for this marketing program would make sense in the long run.  For Q1 or for this test pilot, we might start with some of the personal accounts that we already have, like the ones Piber shared and like he also wrote above: "we might have to set up community accounts but we don't feel the need to set them up just for the sake of doing so but if this is a preference by the community we can set them up for sure!."

**Comment by zachhorn** [comment]
Hi @piber-dev, I appreciate the initiative here. 

As the Head of Marketing for Overclock Labs, I don't believe this proposal is the best first step for contributing to marketing activities. As we discussed briefly, I would prefer to see this structured as a series of bounties that we can experiment with early next year. 

In general, we have an amazing community and ecosystem of contributors who create content across channels in their free time (which is always encouraged). If we're assessing those same activities in a funding context, we need to look closely at quality and performance (especially reach), which — based on this proposal and prior work samples — is not something that I'm confident has been proven just yet.

I would strongly suggest we start with a bounty for Reddit contributions (in early '25), assess the performance, and go from there.

**Reply by piber-dev** [reply]
hello @zachhorn thanks for your review and your second comment. i do wish that we had more time to go over everything during the pre-phase of the discussion and before the end of the year so that we could align better! we worked with some of the feedback from yesterdays meeting with adam who mentioned he had talked a little with you, but we really want to try get started as soon as possible and will be trying to push a variant of the proposal to chain for the community to decide.

we worked with some of the feedback from adam, elaborated on a few bullet points, and changed some of the non-billed items (currently in review from other members and validators, but can be seen publicly [here](https://github.com/piber-dev/marketing-prop/blob/main/README.md) as a new draft.) but please keep in mind that regardless if it passes or not, we are eager to keep working with feedback and discussion with everyone on overclock labs and you are one of the key persons we want to support with this proposal. we don't want to make drastic changes when people have already commented whether or not they like the proposal, so we have only made changes to the non-billable items.

we really believe in a number of initiatives inside this proposal and do not fully agree with all the feedback we have been given from everyone who has given it. for example, we want the marketing budget to exist so we can keep experimenting with new items before they eventually may end up being "core tasks" as scott called them. we agree that we can make some of the line items smaller, such as upwards of $9k to spend on youtube sponsorships, but if we end up spending all of it it was a great call to not make the line item smaller.

**Reply by zachhorn** [reply]
As an example, here are two questions that would need to be answered to make an informed decision on a proposal like this:

1. Regarding posts Medium and X: How many followers do each of your accounts have, and what do you expect the reach of your posts to be? Links to ~10 prior posts (at a minimum) and their engagement data would be needed to make an informed decision here. 

2. Regarding the newsletter: How many email addresses do you have on your list? What is your average open rate?

I would love to figure out how to explore some of these topics via bounties in the new year. But as I said in my original post above, in my opinion — this proposal is not ready to go onchain.

**Comment by piber-dev** [comment]
An update was made to this discussion today (2024-12-20) which elaborates some of the bullet points and changes some experimental/non-billed task items. We will publish this proposal to chain next week and we are hoping the community wants us to have a shot with all the strategies in the proposal, otherwise we will make any required changes and keep trying next year.

**Comment by piber-dev** [comment]
The proposal is live!

https://www.mintscan.io/akash/proposals/276

**Comment by SophoclesSummer** [comment]
This proposal was rejected on 1/2/2025 by a 80% majority. 
https://www.mintscan.io/akash/proposals/276
